server/sonar-web/src/main/js/app/components/nav/global/GlobalNavUser.js

```diff
@@ -45,26 +72,70 @@ class GlobalNavUser extends React.PureComponent {
     }
   };
 
-  handleLogout = e => {
+  handleLogout = (e: Event) => {
     e.preventDefault();
     this.props.router.push('/sessions/logout');
+    this.closeDropdown();
+  };
+
+  toggleDropdown = (evt: Event) => {
+    evt.preventDefault();
+    if (this.state.open) {
+      this.closeDropdown();
+    } else {
+      this.openDropdown();
+    }
+  };
+
+  openDropdown = () => {
+    window.addEventListener('click', this.handleClickOutside);
+    if (this.state.lastLoadedUser !== this.props.currentUser) {
```

Comment:
simply fetch organizations every time

otherwise:

1. open dropdown, see organizations
2. create new organization
3. open dropdown => there is no new organization